### PR TITLE
Updated events

### PR DESCRIPTION
**Pull request by haricm**
Added JuliaCon, Metarefresh, JSFoo, Data Science in R workshop. 
Removed Data Science in Python workshop.

### CODE REVIEW

File: hasweb/templates/index.html [hunk 1]
```diff
@@ -43,12 +43,18 @@
   </div>
   <ul class="cardset row">
     {%- for etitle, edate, eurl, edesc in [
-        ('Introduction to Data Science in Python', 'Sat, Mar 12 – Sun, Mar 13, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-python-1/',
+        ('Introduction to Data Science in R', 'Sat, Apr 2 – Sun, Apr 3, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-r-1/',
           'Ever wondered what it takes to work in the field of Data Science?'),
         ('Rootconf 2016', 'Thu, Apr 14 – Fri, Apr 15, Bangalore', 'https://rootconf.talkfunnel.com/2016/',
           'On learning from failure in DevOps'),
+        ('Metarefresh 2016', 'Fri, May 13 – Sat, May 14, Bangalore', 'http://metarefresh.in/',
```
Comment (jace) [inline hasweb/templates/index.html]: The event's name is `Meta Refresh`

Comment (jace) [inline hasweb/templates/index.html]: The URL should point to Funnel since the website isn't up yet.

File: hasweb/templates/index.html [hunk 2]
```diff
@@ -43,12 +43,18 @@
...
+          'About product stories'),
```
Comment (jace) [inline hasweb/templates/index.html:51]: This caption is incorrect.

File: hasweb/templates/index.html [hunk 3]
```diff
@@ -43,12 +43,18 @@
...
+        ('JuliaCon India 2016', 'Wed, Jul 27, Bangalore', 'http://juliacon.in/',
+          'A conference on Julia language and computing ecosystem'),
         ('The Fifth Elephant 2016', 'Thu, Jul 28 – Fri, Jul 29, Bangalore', 'https://fifthelephant.talkfunnel.com/2016/',
           "India's most renowned data science conference"),
+        ('JSFoo 2016', 'Thu, Sep 15 – Fri, Sep 16, Bangalore', 'http://jsfoo.in/',
```
Comment (jace) [inline hasweb/templates/index.html]: Please link to funnel and don't use `http://` in the URL when `https://` works.

File: hasweb/templates/index.html (hunk 2 above)
Comment (haricm) [inline hasweb/templates/index.html:51]: Just got this tagline from Zainab.

File: hasweb/templates/index.html (hunk 1 above)
Comment (haricm) [inline hasweb/templates/index.html]: We haven't started CfP. That's why I directed it to home page. Shall I change url to https://metarefresh.talkfunnel.com ?

Comment (jace) [inline hasweb/templates/index.html]: We should not list events on the site until the CfP is open. There has to be a clear call to action.

File: hasweb/templates/index.html (hunk 3 above)
Comment (haricm) [inline hasweb/templates/index.html]: We haven't started CfP.

File: hasweb/templates/index.html [hunk 4]
```diff
@@ -43,10 +43,12 @@
   </div>
   <ul class="cardset row">
     {%- for etitle, edate, eurl, edesc in [
-        ('Introduction to Data Science in Python', 'Sat, Mar 12 – Sun, Mar 13, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-python-1/',
+        ('Introduction to Data Science in R', 'Sat, Apr 2 – Sun, Apr 3, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-r-1/',
           'Ever wondered what it takes to work in the field of Data Science?'),
         ('Rootconf 2016', 'Thu, Apr 14 – Fri, Apr 15, Bangalore', 'https://rootconf.talkfunnel.com/2016/',
           'On learning from failure in DevOps'),
+        ('Meta Refresh 2016', 'Fri, May 13 – Sat, May 14, Bangalore', 'https://metarefresh.talkfunnel.com/2016/',
+          'About product stories'),
```
Comment (jace) [inline hasweb/templates/index.html:51]: Meta Refresh 2016 CfP is not public yet.

**Pull request merged**